src/driver.ts

```diff
@@ -731,6 +731,7 @@ export abstract class CMakeDriver implements vscode.Disposable {
         return [
           '/m',
           '/property:GenerateFullPaths=true',
+          '/consoleloggerparameters:NoSummary'
```

Comment:
Hard coded build arguments were removed in a recent PR, since it can cause issues.

Reply:
Done